Answer drag to textbox

dltn opened the pull request:
![ansdnd](https://cloud.githubusercontent.com/assets/6599399/25114581/8eeee46e-23cd-11e7-8006-8c57ed9f419d.gif)

Allows you to drag search result snippets directly into answer textboxes. Graphically, the search result floats back to its original position after appending its content to the text area.

tylermzeller committed:
Change nlc classifier

dltn committed:
added doc convert tests

dltn committed:
added rr tests

dltn committed:
added nlc tests

dltn committed:
fix word doc body

dltn committed:
preparation refactor

3ygun committed:
Partial MC choice extraction

"Works" if the question's options are the following question prompt
after doc_conversion (ex: 3-questions.pdf). BUT fails if the options are
within a single line...

For testing one will need to reclassify the MC question on
3-questions.pdf from TF to MC to see the change

3ygun committed:
Fixes the majority of Multiple Choice parsing issue

- Questions choices that are appended to the prompt
- Questions choices that are held in the following questions

3ygun committed:
Fixed the conjoined questions problem

- Resolved multiple questions being within a single question
- Added ability to have captal ABCD and : deliminated choice questions
- Updated debug messages & other issues

3ygun committed:
Debug message styling

3ygun committed:
Another Example PDF

tylermzeller committed:
Add homeworks from chapter 1 and chapter 2

tylermzeller committed:
Add homeworks and license

3ygun committed:
Removed unused NLC endpoint

Isn't in use, so remove

3ygun committed:
Converted PDFUploadDemo to ES6

Matching the rest of the project and extracted elements of the render
methods for readibility

3ygun committed:
Added Classifier Selector

Want to switch between the classifiers? For now that is an option!

dltn committed:
Add testing info to README

3ygun committed:
Fixed NLC API Issue when using new classifier

Fixes:
```
Not Found: /api/upload/questions/90e7b4x199-nlc-32429/
[17/Apr/2017 17:17:36] "POST /api/upload/questions/90e7b4x199-nlc-32429/
HTTP/1.1" 404 4943
```
^ It wasn't picking up the entire classifier name I guess

dltn committed:
progress

dltn committed:
answer dnd to textbox

dltn committed:
Merge branch 'master' of https://github.com/signofthehorns/watson-ta

dltn committed:
merge resolve

dltn committed:
cleanup debug artifacts

3ygun committed:
Resolved Webpack-config.js formatting issues

3ygun commented:
Some CSS work is needed for usability. I would be fine okaying it now as it meets base requirements however usability issues like

- [x] Drag to questions below the Knowledge base box etc. aren't good. (Will work on before the demo a quick prototype seemed promising)

3ygun committed:
Usability UI Changes

- Hid Messenger Button, Solr Info and Left Side Menu
- Fixed the position of the Knowledge Base to always be on screen

3ygun commented:
I made some UI/CSS tweeks to better showoff the product. Including:

- [x] Hid Messenger Button, Solr Info and Left Side Menu -- None of this is user relevant content currently
- [x] Fixed the position of the Knowledge Base to always be on screen -- Our killer lazy feature

Now possible to drag to questions anywhere in the page:

![watsonta--drag-fixed-position](https://cloud.githubusercontent.com/assets/5482320/25149136/5ff92614-244b-11e7-8d12-a6f250f3333c.PNG)

3ygun committed:
Merge branch 'master' into answer-drag
